app/views/admin/groups/_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= simple_form_for([:admin, group]) do |f| %>
+  <%= f.input :name %>
+  <%= f.input :number %>
+  <%= f.association :introduction, collection: Introduction.all, label_method: :to_s %>
+  <%= f.association :users, collection: User.all, input_html: { class: 'select2' },
```

Comment:
`User.by_firstname.member.confirmed`

and

remove space inside the hash (I think).